### ISSUE TITLE
[vstest]Remove config calls from test_virtual_chassis.py in swss vs tests

### ISSUE THREAD

**Issue by dgsudharsan** [sonic-net/sonic-buildimage#10246]
#### Description
As a part of ongoing effort to decouple vstests from sonic-utilities, the click commands called in vs tests should be modified to redis-CLI based config_db set.
This is to remove vs tests from running in sonic-utilities pipeline checkers

Example of existing command in test_virtual_chassis:
https://github.com/Azure/sonic-swss/blob/00266891a9f41e5bd37100b05237e865cde429b2/tests/test_virtual_chassis.py#L40

Example of how to migrate:
https://github.com/Azure/sonic-swss/pull/2179

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by abdosi** [comment]
@saksarav-nokia : Please help in updating test to new infra.

**Comment by saksarav-nokia** [comment]
@abdosi, This issue is fixed in PR #2214 .
